Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/VaccinatedPerson.kt

```diff
@@ -0,0 +1,32 @@
+package de.rki.coronawarnapp.vaccination.core
+
+import org.joda.time.Instant
+import org.joda.time.LocalDate
+
+data class VaccinatedPerson(
+    val vaccinationCertificates: Set<VaccinationCertificate>,
+    val proofCertificates: Set<ProofCertificate>,
+    val isRefreshing: Boolean,
```

Comment:
What does refreshing mean in this context?

Reply:
That this data is currently being updated, e.g. polling servers for a new ProofCertificate.
Use case:
User presses refresh button in UI, UI wants to display a progress spinner.
This can't be only handled on viewmodel level, as there will also be automatic refreshes/updates via workers, that could overlap with the user having the UI open.

Reply:
Can we call it isUpdatingData or the like to make more clear? My first association was refreshing the vaccination.

Reply:
Yes, will do in another PR.